install-components.md

```diff
@@ -1622,88 +1622,122 @@ To install Supply Chain Security Tools - Store:
+[cosign quick start guide](https://github.com/sigstore/cosign#quick-start) to
+generate your own keys and sign an image.
+
+### Installation
 
 To install Supply Chain Security Tools - Sign:
 
 1. List version information for the package by running:
 
-    ```bash
+    ```shell
     tanzu package available list image-policy-webhook.signing.run.tanzu.vmware.com --namespace tap-install
     ```
     For example:
 
-    ```bash
+    ```shell
     $ tanzu package available list image-policy-webhook.signing.run.tanzu.vmware.com --namespace tap-install
     - Retrieving package versions for image-policy-webhook.signing.run.tanzu.vmware.com...
       NAME                                               VERSION         RELEASED-AT
       image-policy-webhook.signing.run.tanzu.vmware.com  1.0.0-beta.1    2021-10-25T00:00:00Z
+      image-policy-webhook.signing.run.tanzu.vmware.com  1.0.0-beta.2    2021-11-29T00:00:00Z
     ```
 
-2. (Optional) To make changes to the default installation settings, run:
-
-    ```bash
-    tanzu package available get image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.1 --values-schema --namespace tap-install
+1. (Optional) To make changes to the default installation settings, run:
+    ```shell
+    tanzu package available get image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.2 --values-schema --namespace tap-install
     ```
     For example:
 
-    ```bash
-    $ tanzu package available get image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.1 --values-schema --namespace tap-install
-    | Retrieving package details for image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.1...
+    ```shell
+    $ tanzu package available get image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.2 --values-schema --namespace tap-install
+    | Retrieving package details for image-policy-webhook.signing.run.tanzu.vmware.com/1.0.0-beta.2...
       KEY                     DEFAULT  TYPE     DESCRIPTION
       allow_unmatched_images  false    boolean  Feature flag for enabling admission of images that do not match
                                                 any patterns in the image policy configuration.
                                                 Set to true to allow images that do not match any patterns into
                                                 the cluster with a warning.
-    ```
+      quota.pod_number        <nil>    string   The maximum number of Image Policy Webhook Pods allowed to be
+                                                created with the priority class system-cluster-critical. This
+                                                value must be enclosed in quotes (""). If this value is not
+                                                specified then the default value of 5 is used.
+      replicas                <nil>    integer  The number of replicas to be created for the Image Policy
```

Comment:
Should the default column show 5 and 1 respectively? I find confusing that the default column says <nil>, but description mentions a default value.

Reply:
Good catch too!